docs/docs/configuration.md

```diff
@@ -204,6 +204,24 @@ For example:
 # In this example, the following will not be packaged in to MTA archive:
 # all files with the"txt" extension and the "mtaignore" folder within the "myfolder" directory. 
      
+```
+From MBT v1.2.25 version, build-parameters ignore attribute will be upgraded to support [Full Glob Pattern](https://en.wikipedia.org/wiki/Glob_(programming)).
+
```

Comment:
How can it be that ! is used to match and to negate?
 
 yes, we can.

By using negation pattern `!` in the `build-parameters ignore` attribute, because cause `not ignore` equal to `include`, you can select specified content into the package.

Reply:
Ok.
By using the `!` negation pattern in the `build-parameters ignore` attribute, because `not ignore` equals `include`, you can select specified content into the package.